### PR TITLE
Update CHANGES.md

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
fixed typo

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 * Added 16 new locales for locale service and email templates (af, ar, bn, cz, hu, hy, jv, lt, no, ru, si, sv, ta, vi, zh-cn, zh-tw)
 * New users service routes to allow updates pref and name update
-* 2 stage Docker build
+* 2 stages Docker build
```
Comment (eldadfux) [inline CHANGES.md]: 2 stage build is the name of the feature.

Comment (sakshamtaneja21) [reply CHANGES.md]: sure

Comment (sakshamtaneja21) [reply CHANGES.md]: have changed

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳 
Please reach out to me on our [Discord server](https://discord.gg/GSeTUeA) if you would like me to send you some cool Appwrite stickers and magnets!

To say thank you, we have just invited you to join the Appwrite organization on GitHub. Accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership visibility settings, so your new Appwrite team membership will show up on your personal GitHub profile.

**Pull request merged**